package.json

```diff
@@ -10,7 +10,7 @@
     "micro-services"
   ],
   "dependencies": {
-    "@hyperswarm/dht": "^3.1.0",
+    "@hyperswarm/dht": "git+https://github.com/hyperswarm/dht.git#adaptive-ephemerality",
```

Comment:
TODO: update to release version when https://github.com/hyperswarm/dht/pull/27 lands